core/sql/exp/exp_function.cpp

```diff
@@ -786,13 +786,18 @@ ExpRaiseErrorFunction::ExpRaiseErrorFunction (Attributes **attr,
 					      NABoolean raiseError,
 					      const char *constraintName,
 					      const char *tableName,
-						  const NABoolean hasStringExp)  // -- Triggers
+                                              const NABoolean hasStringExp,  // -- Triggers
+                                              const char * optionalStr)
 : ex_function_clause (ITM_RAISE_ERROR, (hasStringExp ? 2 : 1), attr, space),
-    theSQLCODE_(sqlCode),
-    constraintName_((char *)constraintName),
-    tableName_((char *)tableName)
+  theSQLCODE_(sqlCode),
+  constraintName_((char *)constraintName),
+  tableName_((char *)tableName)
 {
-    setRaiseError(raiseError);
+  setRaiseError(raiseError);
+
+
+  strncpy(optionalStr_, optionalStr, MAX_OPTIONAL_STR_LEN);
```

Comment:
This will abend if "optionalStr" is null

Reply:
thanks, good catch. this showed up during regr failures as well.
It is fixed.